nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestBulkProcessor.java

```diff
@@ -244,10 +244,6 @@ public void testSequentialCommand() throws Exception {
         assertFalse(overlapCommands(
                 results.stream().filter(r -> "dummySequential".equals(r.getAction())).collect(Collectors.toList()),
                 true));
-        // concurrent commands must overlap
```

Comment:
we are submitting a concurrent command above so it is weird to do nothing with it, we can let a comment like:
// Concurrent commands should overlap but not everytime, so it cannot be tested in a reliable way